alerter.cpp

```diff
@@ -13,7 +22,13 @@ int networkAlertStub(float celcius) {
 
 void alertInCelcius(float farenheit) {
     float celcius = (farenheit - 32) * 5 / 9;
-    int returnCode = networkAlertStub(celcius);
+	int returnCode;
+	if(AlertTest){

```

Comment:
Production code should not be aware of test code. Test code should be abstracted neatly via an interface or a function pointer.
There is a danger of accidental execution of test code in field!!